### ISSUE TITLE
Connexion jwt support

### ISSUE THREAD

**Issue by ctippur** [zalando/connexion#607]
### Description

Wondering if there is support for jwt auth via connexion

### Expected behaviour

Should be a way to authenticate jwt token, I am unable to find a way to initialize jwt token via the flask app.

### Actual behaviour

I get an error:
```
raise RuntimeError("You must initialize a JWTManager with this flask "
RuntimeError: You must initialize a JWTManager with this flask application before using this method
```

I see an issue reported earlier but I was not sure if it led to a conclusion.

https://github.com/zalando/connexion/issues/124

Appreciate if I could get a working example of the same (if any)

### Additional info:

Output of the commands:

- `python --version`
3.6.2
- `pip show connexion | grep "^Version\:"`
Version: 1.1.9

**Comment by bobh66** [comment]
Did you try flask_jwt_extended:

```
from flask_jwt_extended import JWTManager

app.config['JWT_SECRET_KEY'] = 'jwt-secret-string'

jwt = JWTManager(app)
```

**Comment by ctippur** [comment]
Yes. I tried to use that.
I get an error trying to use the below line of code:
```access_token = create_access_token(identity=username)```
```
raise RuntimeError("You must initialize a JWTManager with this flask "
RuntimeError: You must initialize a JWTManager with this flask application before using this method
```

**Comment by ctippur** [comment]
I am using the below code in my controller

```
pp=connexion.FlaskApp(__name__).app
# Setup the Flask-JWT-Extended extension
app.config['JWT_SECRET_KEY'] = 'super-secret'  # Change this!
app.config['JWT_TOKEN_LOCATION'] = ['cookies','header']
jwt = JWTManager(app)
```

**Comment by dtkav** [comment]
JWT auth was introduced with #732 and is available in connexion 2.0.x
